docs/help/bundle.md

```diff
@@ -88,7 +88,7 @@ metadata:
   namespace: operators
 spec:
   sourceType: grpc
-  image: quay.io/hgao/operator-catalog:v0.0.1
+  image: quay.io/hgao/operator-catalog:latest
```

Comment:
this doesn't make sense. You can update the doc as @brusdev suggested. and referece to **hgao** should be replaced by a common name to indicate any repo of user's choice. something lik quay.io/<your repo>/....